### ISSUE TITLE
ValueError: Unknown format code 'r' for object of type 'str'

### ISSUE THREAD

**Issue by MinchinWeb** [mx-moth/django-enumchoicefield#17]
```
File "[...]\.venv\lib\site-packages\enumchoicefield\fields.py", line 69, in get_prep_value
    raise ValueError("Unknown value {value:r} of type {cls}".format(
  File "C:\Python310\lib\enum.py", line 781, in __format__
    return cls.__format__(val, format_spec)
ValueError: Unknown format code 'r' for object of type 'str'
```

Seems it can't turn a string into something to feed into the database?

Django 4.0.4
django-enumchoicefield 3.0.0

**Comment by MinchinWeb** [comment]
Turns out I was feeding the wrong enum into the field, but the error message is still a separate issue...

**Comment by mx-moth** [comment]
Thanks for the report. I've fixed this, added tests, and released it as version 3.0.1
